Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -174,7 +124,7 @@ - (void)updateBlockNotification:(SNTStoredEvent *)event withBundleHash:(NSString
       self.event.fileBundleHash = bundleHash;
       [self.foundFileCountLabel removeFromSuperview];
       [self.hashingIndicator setHidden:YES];
-      [self.openEventButton setEnabled:YES];
+      // [self.openEventButton setEnabled:YES];
```

Comment:
Is this meant as another reminder for future work or should this be removed?

Reply:
It's a reminder that the button should be disabled while hashing is in progress and re-enabled afterward